mailpoet/tests/integration/Migrations/Migration_20230712_180341_Test.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types = 1);
+
+namespace integration\Migrations;
```

Comment:
As far as I can tell it doesn't matter, but the one other migration test we have uses the `MailPoet\Migrations` namespace. Is there a reason you went with this instead?

Reply:
My best guess is that this was autogenerated by PHPStorm and I didn't notice. I don't I added this. I just pushed a commit changing the namespace.